ocaml/xapi/qcow_tool_wrapper.ml

```diff
@@ -0,0 +1,65 @@
+(*
+ * Copyright (C) 2025 Vates.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
```

Comment:
I think it'd be easier to review (and revert in case of errors) if you introduced the qcow_tool_wrapper in a separate commit, and only then started using it import_raw_vdi etc.

Reply:
Separate commit not separate PR right?

Reply:
yes